platform/iphone/detect.py

```diff
@@ -147,8 +147,11 @@ def configure(env):
     if (env["target"].startswith("release")):
 
         env.Append(CPPFLAGS=['-DNDEBUG', '-DNS_BLOCK_ASSERTIONS=1'])
-        env.Append(CPPFLAGS=['-O2', '-flto', '-ftree-vectorize', '-fomit-frame-pointer', '-ffast-math', '-funsafe-math-optimizations'])
-        env.Append(LINKFLAGS=['-O2', '-flto'])
+        env.Append(CPPFLAGS=['-O2', '-ftree-vectorize', '-fomit-frame-pointer', '-ffast-math', '-funsafe-math-optimizations'])
+        env.Append(LINKFLAGS=['-O2'])
+        if env['use_lto'] == 'yes':
```

Comment:
It's not defined here, so that won't work. It works for X11 and Windows because it's defined as platform-specific options.

Reply:
Also I think it makes sense to enable it by default for export templates, as it improves performance a lot. The only drawback is that it makes incremental builds pretty moot.

Reply:
I tested it and it works as expected.

Reply:
That's only by chance, because we now use `BoolVariable('use_lto', 'Use link time optimization', False),` `platform/x11/detect.py`, which registers the variable at the global level and no longer just for that platform. It should be moved to `SConstruct` and removed from `platform/windows/detect.py`, then you can use like you do for iphone.